### PR TITLE
Added pre-commit hook for kacl-verify

### PR DESCRIPTION
**Pull request by alfechner**
This is a configuration file for [pre-commit](https://pre-commit.com/).

It enables pre-commit to use `kacl-verify` on a repository before committing new code.

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- id: kacl-verify
+  name: "verify kacl/changelog"
+  description: 'Veryfies if the changelog is in "keep-a-changelog" format.'
```
Comment (Qu4tro) [inline .pre-commit-hooks.yaml]: *Verifies

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  entry: kacl-cli verify
+  language: python
+  types: [python]
```
Comment (Qu4tro) [inline .pre-commit-hooks.yaml]: I think we only need to verify `CHANGELOG.md`, so instead of `types: [python]`, I think it should be `files: '^CHANGELOG.md'`

Comment (mschmieder) [reply .pre-commit-hooks.yaml]: @alfechner this is great, thanks for the PR. Could you please change the hook according to @Qu4tro request?

**Comment by Qu4tro** [comment]
Hey. Small note:

`always_run` could be set to True and be perfectly valid. This is because, even if the CHANGELOG.md is not changed, `.kacl.yml` might have. `always_run` as False, leave us with better performance as it won't ever be triggered unnecessarily, but some false negatives might pop up. I think, as is, is a good compromise, nevertheless. Users can set `always_run = true` if they want the other side of the tradeoff and if too many people raise this issue we can always add it, as it's a trivial change.

TLDR: Looks good to me.
